### DISCUSSION TITLE
这个东西是不是最好在GPU容器里面跑啊? 为了llama?

### DISCUSSION THREAD

**Discussion by ca1123** [GaiZhenbiao/ChuanhuChatGPT#651] [General]
[https://github.com/GaiZhenbiao/ChuanhuChatGPT/blob/main/requirements_advanced.txt](url)

transformers
torch
icetk
protobuf==3.19.0
git+https://github.com/OptimalScale/LMFlow.git
cpm-kernels
sentence_transformers

**Comment by GaiZhenbiao** [comment]
是的，这些是运行本地模型需要的。
